demos-europe/demosplan-core
# bug(refs DPLAN-12623): Fix adding and editing places

### Ticket
DPLAN-12623


- Add Checks if the given place name is unique within the list of places. To update an existing place the currentPlaceId is given and the place with this id is excluded from the check.
- Add error handling for update request
- Add id from request to new places to fix sorting

### How to review/test
Verfahren -> Übersicht -> Bearbeitungsschritte
- Add a new place -> Sort it -> Reloade page -> Sorting should be saved 
- Edit a place -> Take same name as another place -> Saving should throw an error explaining that you cannot have the same name
